src/SearchResult.tsx

```diff
@@ -3,10 +3,10 @@ import { DeserializedSearchResult } from "./types";
 type Props = {
   searchResult: DeserializedSearchResult;
   isSelected?: boolean;
-  shouldOpenInNewWindow?: boolean;
+  shouldOpenInNewTab?: boolean;
 };
 
-export const SearchResult = ({ searchResult, isSelected = false, shouldOpenInNewWindow = false }: Props) => {
+export const SearchResult = ({ searchResult, isSelected = false, shouldOpenInNewTab = false }: Props) => {
```

Comment:
Nit: I think `openInNewTab` or `opensInNewTab` would be more expressive -- conveying the same meaning, but terser.

Reply:
*That's* the wording I've been searching for!

Reply:
https://github.com/vectara/react-search/pull/22/commits/8711b05a1fd7913806accfe756898908baeaf09f